docs/getting-started/full-dotnet/sample/EFGetStarted.ConsoleApp.ExistingDb/ScaffoldCommand.txt

```diff
@@ -0,0 +1 @@
+﻿Scaffold-DbContext -provider EntityFramework.MicrosoftSqlServer -connection "Server=(localdb)\mssqllocaldb;Database=Blogging;Trusted_Connection=True;"
```

Comment:
BTW I put this in a file rather than an inline code block because you don't get scroll bars for inline stuff (the content just gets hidden off the side of the page). I didn't want to split it on to multiple lines, since then you can copy/paste very well.